Gruntfile.coffee

```diff
@@ -1,85 +0,0 @@
-module.exports = (grunt) ->
```

Comment:
No one is proactively supporting the [`grunt-protractor-runner`](https://www.npmjs.com/package/grunt-protractor-runner) project, so I decided to try ripping out Grunt altogether. The scripts necessary to create the same functionality were not too difficult, so I'm advocating for just managing the build process using `npm`.